runtime/compiler/optimizer/VectorAPIExpansion.cpp

```diff
@@ -1666,13 +1666,78 @@ TR_VectorAPIExpansion::transformIL(bool checkBoxing)
          continue;
 
       bool scalarized;
-      bool vectorizedNode;
-
+      bool vectorizedOrScalarizedNode;
       TR::DataType elementType;
       int32_t bitsLength;
       vapiObjType objectType;
 
-      bool vectorizedOrScalarizedNode = isVectorizedOrScalarizedNode(node, elementType, bitsLength, objectType, scalarized);
+      vectorizedOrScalarizedNode = isVectorizedOrScalarizedNode(node, elementType, bitsLength, objectType, scalarized);
+
+      // Vectorize intrinsic if its operands are known
+      if (boxingAllowed() &&
+         !vectorizedOrScalarizedNode &&
+         opCode.isFunctionCall())
+         {
+         TR::MethodSymbol * methodSymbol = node->getSymbolReference()->getSymbol()->castToMethodSymbol();
```

Comment:
There are various styles of pointer variable declaration in this file, such as `TR::Node *node = ...;`, `TR::MethodSymbol * methodSymbol = ...;`, and `TR::Node* operand = ...;`.  Please use the same style, i.e. where to put the whitespace and `*`.